src/components/common/CupIcon.jsx

```diff
@@ -0,0 +1,22 @@
+const CupIcon = ({src, alt}) => {
+    return (
+        <img 
```

Comment:
Next使ってるのでNextの提供しているImageコンポーネントが良いかもしれないです。
https://nextjs.org/docs/api-reference/next/image

Reply:
That is good to know! Thank you for sharing!

Can I add _width_ and _height_ attributes with arbitrary pixels?
The article says those 2 attributes are required except for statically imported images. Still, I'm not planning to import images statically in this component since it passes the image paths depending on its props.

Reply:
Yes you can add width and height as props then we can specify them when use them.

Reply:
I replaced the img tag with Next/image.
I also added width and height for props whose default values are 100.